volunteer.php

```diff
@@ -121,19 +121,33 @@ function volunteer_civicrm_navigationMenu(&$params) {
         ),
         $newNavId + 5 => array(
           'attributes' => array(
+            'label' => ts('Configure Default Project Settings', array('domain' => 'org.civicrm.volunteer')),
+            'name' => 'volunteer_config_defaults',
+            'url' => 'civicrm/volunteer/defaults',
```

Comment:
If we add defaults for other entities, such as Needs, would those appear here as well? If not, the URL should specify "project."